cromwell-helm/templates/cromwell.yaml

```diff
@@ -166,6 +160,48 @@ data:
             }
           }
         }
+        GCPBATCH {
+          actor-factory = "cromwell.backend.google.batch.GcpBatchBackendLifecycleActorFactory"
+          config {
+            // Google project
+            project = "{{ .Values.config.gcsProject }}"
+
+            // Base bucket for workflow executions
+            root = "{{ .Values.config.gcsBucket }}"
+
+            // Polling for completion backs-off gradually for slower-running jobs.
+            // This is the maximum polling interval (in seconds):
+            maximum-polling-interval = 600
+
+            batch {
+              // A reference to an auth defined in the `google` stanza at the top.  This auth is used to create
+              // Batch Jobs and manipulate auth JSONs.
+              auth = "application-default"
+
+              // alternative service account to use on the launched compute instance
+              // NOTE: If combined with service account authorization, both that service account and this service account
+              // must be able to read and write to the 'root' GCS path
+              compute-service-account = "{{ .Values.config.serviceAccount.annotations.gcpServiceAccount }}"
+
+              // Location to submit jobs to Batch and store job metadata.
+              location = "{{ .Values.config.gcsRegion }}"
+            }
+
+            filesystems {
+              gcs {
+                // A reference to a potentially different auth for manipulating files via engine functions.
+                auth = "application-default"
+                // Google project which will be billed for the requests
+                project = "{{ .Values.config.gcsProject }}"
+              }
+            }
+            // Cromwell will replace * with the correct region for each job submission.
```

Comment:
@aednichols and/or @salonishah11  happy to chat tomorrow, but this does not seem to be the case. When trying to test this I am getting this very straightforward error:

``` Unable to complete Batch request due to a problem with the request (io.grpc.StatusRuntimeException: INVALID_ARGUMENT: subnetwork field is invalid. subnetwork: projects/terra-quality-e438a884/regions/*/subnetworks/subnetwork is not matching the expected format: regions/([a-z0-9-_]+)/subnetworks/([a-z]([-a-z0-9]*[a-z0-9])?)$). 
```

Reply:
In this case, we likely want to replace the `*` with a {{ .Values.config.gcsRegion }}

Reply:
Actually, this might instead mean that the cromwell chart needs to point to the latest Cromwell release 🤔

Reply:
Yep, I think I need to update this: https://github.com/broadinstitute/cromwhelm/blob/0f5e53b8488e0fa2f2614826920635af780f57dc/terra-batch-libchart/values.yaml#L5

Reply:
Things are now working when pointing to cromwell:89-561bbf4